test/integration/create-cluster-on-hub.bats

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function token_cleanup() {
+	curl -X DELETE https://discovery-stage.hub.docker.com/v1/clusters/$1
```

Comment:
I'm not sure if we're going to get the `swarm destroy` soon enough and we need this integration test.

For me it's an LGTM

@abronan @vieux Thoughts?